onnxruntime/core/graph/graph.cc

```diff
@@ -984,6 +945,52 @@ bool Node::ClearAttribute(const std::string& attr_name) {
   graph_->SetGraphProtoSyncNeeded();
   return attributes_.erase(attr_name) > 0;
 }
+
+void Node::ToProto(NodeProto& proto, bool
```

Comment:
Just wondering why github is showing the whole function body as changed, when it is only a few lines that have changed. Oh, well.

Reply:
It was moved.